data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Original:</p><p>Running the following cell will create a scaler using the MinMaxScaler from scikit-learn and fit the gas prices to the scaler.</p><p> </p><p>Proposed:</p><p>Running the following cell will create a scaler using the MinMaxScaler from scikit-learn and <strong>then </strong>fit the gas prices to the scaler.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Done</p>